### PR TITLE
Improved Looks

### PR DESCRIPTION
**Pull request by QuiteAFancyEmerald**
Added support for the theme in the friends tab which still uses default Discord stuff (which is a pretty major section grayed out)

Regardless though this theme is pretty epic. :)

Original:
image

Changes:
image

### CODE REVIEW

**Comment by Ryan-da-ghost** [comment]
Nice !

**Comment by Ryan-da-ghost** [comment]
I merged The Great new Update :) I'll Hopefully Fix these issues in the new Updates ! 👍

**Comment by QuiteAFancyEmerald** [comment]
👍

File: css/discord.css [hunk 1]
```diff
@@ -1,126 +1,109 @@
...
+}
+#PENDING-tab {
+    background-color: var(--textinputs);
 }
-
-.lookFilled-1Gx00P.colorGreen-29iAKY :hover {
-  color: #fff;
-  background-color: var(--btn-color);
+#BLOCKED-tab {
+    background-color: var(--textinputs);
```
Comment (Ryan-da-ghost) [inline css/discord.css:108]: Make it background-color: var(--textinputs) !important

File: css/discord.css [hunk 2]
```diff
@@ -1,126 +1,109 @@
...
+    background-color: var(--btn-color);
+}
+#ONLINE-tab {
+    background-color: var(--textinputs);
+}
+#ALL-tab {
+    background-color: var(--textinputs);
+}
+#PENDING-tab {
+    background-color: var(--textinputs);
```
Comment (Ryan-da-ghost) [inline css/discord.css:105]: Make it background-color: var(--textinputs) !important

File: css/discord.css [hunk 3]
```diff
@@ -1,126 +1,109 @@
...
-  background-color: var(--btn-color);
+    color: #fff;
+    background-color: var(--btn-color);
+}
+.lookFilled-1Gx00P.colorGreen-29iAKY:hover {
+    color: #fff;
+    background-color: var(--btn-color);
+}
+#ONLINE-tab {
+    background-color: var(--textinputs);
```
Comment (Ryan-da-ghost) [inline css/discord.css:99]: Make it background-color: var(--textinputs) !important

**Review by Ryan-da-ghost — COMMENTED** [review]
!important tag requested on some of CSS styles

**Comment by QuiteAFancyEmerald** [comment]
Made another pull request with those changes plus some more edits for improving Discord support. :)

**Pull request merged**